docs/ums/README.md

```diff
@@ -176,12 +261,32 @@ There are two ways to change the password for a user by using their old password
 
 ### Using Their Password
 
-<CodeSwitcher :languages="{js:'JavaScript',bash:'cURL'}">
+<CodeSwitcher :languages="{js:'JavaScript',py:'Python',bash:'cURL'}">
 <template v-slot:js>
 
 ``` js
-ums.changePassword('Elon@tesla.com', oldPassword, newPassword); 
+ums
+.changePassword('Elon@tesla.com', oldPassword, newPassword)
+.subscribe(user => {}, error=>{});   
 ```
+</template>
+<template v-slot:py>
+
+``` py
+...
```

Comment:
I would suggest to use full example of code everytime, because some users can open this part of documentation by directly link. And in this case they need to find another part of code.

Reply:
next ver